src/lib.ml

```diff
@@ -67,6 +67,14 @@ module Error = struct
       }
   end
 
+  module Multiple_implementations_for_virtual_lib = struct
+    type t =
+      { lib             : Lib_info.t
+      ; given_variants  : Variant.Set.t
```

Comment:
Ideally, these should come with locations.

Reply:
I added some location in the error but I'm not sure which locations you mean.